genai-perf/genai_perf/config/input/config_command.py

```diff
@@ -196,26 +219,6 @@ def _check_output_format_and_generate_plots(self) -> None:
                     f"User Config: generate_plots is not supported with the {self.endpoint.output_format} output format"
                 )
 
-    def _check_payload_input(self) -> None:
-        if self.input.prompt_source != PromptSource.PAYLOAD:
-            return
-
-        if self.perf_analyzer.get_field("stimulus").is_set_by_user:
-            for key in self.perf_analyzer.stimulus.keys():
-                if key in ["concurrency", "request_rate"]:
-                    raise ValueError(
-                        f"User Config: perf_analyzer.stimulus: {key} is not supported with the payload input source."
-                    )
-
-        if (
-            self.perf_analyzer.measurement.get_field("mode").is_set_by_user
-            and self.perf_analyzer.measurement.mode
-            == PerfAnalyzerMeasurementMode.REQUEST_COUNT
-        ):
-            raise ValueError(
-                f"User Config: perf_analyzer.measurement.mode of request_count is not supported with the payload input source."
-            )
-
```

Comment:
The ShareGPT support added by this PR adds support using payload mode with concurrency or request rate mode.

No need to error on this combination anymore.